torch/nn/modules/conv.py

```diff
@@ -233,8 +233,8 @@ class Conv1d(_ConvNd):
     """.format(**reproducibility_notes, **convolution_notes) + r"""
 
     Shape:
-        - Input: :math:`(N, C_{in}, L_{in})`
-        - Output: :math:`(N, C_{out}, L_{out})` where
+        - Input: :math:`(N, C_{in}, L_{in})` or :math:`(C_{in}, L_{in})`
```

Comment:
We don't have a syntax to say that the `N` dimension is optional?

Reply:
We discussed using `[N]` but went forward with this throughout as it's more explicit without the need for a legend.

I could be convinced otherwise, but we'd need to make consistent changes across modules and add the syntax to [this guide](https://github.com/pytorch/pytorch/wiki/torch.nn-Module-Documentation-Style-Guide).

Imo this doesn't look too bad with a description inline:
``` Input: :math:`([N], C_{in}, L_{in})`, where :math:`[N]` is an optional batch dimension ```

Reply:
I would personally vote for the `[N]` notation as it is shorter if we can use it consistently.